packages/patternfly-4/react-core/package.json

```diff
@@ -27,7 +27,8 @@
   "dependencies": {
     "@patternfly/react-icons": "^2.5.1",
     "@patternfly/react-styles": "^2.3.0",
-    "exenv": "^1.2.2"
+    "exenv": "^1.2.2",
+    "react-transition-group": "1.x"
```

Comment:
Should probably avoid using the 1.x version move to the 2.x since it has been stable for a while.

Reply:
this is an exciting one to get started for many of us... kudos for that! Curious how you arrived on this lib? I have seen a lot of great animation libraries for React, but my favorite so far is probably PopMotion (extended by Pose for declarative animation styles for React) .

https://github.com/Popmotion/popmotion/tree/master/packages/popmotion-pose

At any rate, here is some nifty comparisons to consider if we intend to extend Core animations with React:
https://logrocket.com/blog/comparing-the-top-js-animation-libraries-for-use-in-react-apps-2/

Reply:
I say this having no prior animation design or a11y research on this - this is obviously a bigger topic 😄